errors/src/errors/type_checker/type_checker_error.rs

```diff
@@ -685,6 +685,34 @@ create_messages!(
         help: None,
     }
 
+    @formatted
+    stubs_can_only_have_records_and_transitions {
```

Comment:
I suspect we'll also want stubs to have their respective mappings.

Reply:
That makes sense

Reply:
This would be for the future though as currently there is no way to read external mapping